lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_dependency_installer.rb

```diff
@@ -45,14 +51,15 @@ def install!
             end
             # Wire up all pod target dependencies to aggregate target.
             aggregate_target.pod_targets.each do |pod_target|
+              is_local = sandbox.local?(pod_target.pod_name)
```

Comment:
no need to calculate that unless you are within the `else` branch?

Reply:
True